frontend-styleguide/README.md

```diff
@@ -201,6 +201,56 @@ const Button = (props: PropsWithChildren<ButtonProps>): React.ReactElement => {
 
 <br />
 
+### 👉 Avoid using `&&` on conditional renders
+
+It's a common practice to use `&&` to conditionally renders things inside React components. The inadverted use of this pattern can lead to some unexpected errors [already caught in our application](https://github.com/GeekHunter-Brasil/geekhunter-frontend/pull/752#discussion_r874185216). 
+
+It's a great idea to use ternary operators instead. For further reference, you can check [this article by Kent C. Dodds](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)
+
+❌ Bad
+
+```tsx
+<Box>
+  {errorMessage && <Text>{errorMessage}</Text>}
+</Box>
+```
+
+✅ Good
+
+```tsx
+<Box>
+  {errorMessage ? <Text>{errorMessage}</Text> : null}
+</Box>
+```
+
+<br />
+
+### 👉 Avoid the unnecessary use of `lodash`
+
+It's a common practice to use `lodash` functions along with regular javascript.
+
+`lodash` relies heavier computations than regular javascript, and the library use makes the total size of our application bundle much heavier.
+
+If possible, replace `lodash` functions with regular Javascript.
+
+❌ Bad
+
+```tsx
+<Box>
+  {errorMessage && <Text>{errorMessage}</Text>}
```

Comment:
Missing `lodash` examples

Reply:
One of the rules I intended to write was to avoid `lodash` at all. Most of its operations can be performed with regular javascript and are less computationally heavy. Also, lodash makes the size of our bundle significantly larger. There are some good reads on the subject, [I will leave this one here for reference](https://thejs.dev/jmitchell/its-time-to-let-go-of-lodash-nqc).

On a second look I noticed there is little lodash in the frontend app and thought the rule would be unnecessary.

What do you think?

Reply:
Sorry, this was not intended to be commited.

I wanted to create a rule about avoiding `lodash`, but on a second look I noticed there is little lodash in the frontend app, and thought it would be unnecessary.